<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myCamCon</class>
 <widget class="QMainWindow" name="myCamCon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>myCamCon</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Device:</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="deviceSelection">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>288</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Connect">
        <property name="styleSheet">
         <string notr="true">image: url(:/Images/cam_off.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>458</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cap">
        <property name="styleSheet">
         <string notr="true">background-color:grey;
    border-style: outset;
    border-width:1px;
    border-radius: 10px;
    border-color: beige;
    font: bold 12px;
    min-width: 9em;
    padding: 5px;</string>
        </property>
        <property name="text">
         <string>Capture</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_rec">
        <property name="styleSheet">
         <string notr="true">background-color:grey;
    border-style: outset;
    border-width:1px;
    border-radius: 10px;
    border-color: beige;
    font: bold 12px;
    min-width: 9em;
    padding: 5px;</string>
        </property>
        <property name="text">
         <string>Record</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="cam_Frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="webCamLayout"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
